{
  "id": "openSUSE-SU-2024:10567-1",
  "summary": "python-Pillow-3.1.1-4.1 on GA media",
  "details": "These are all security issues fixed in the python-Pillow-3.1.1-4.1 package on the GA media of openSUSE Tumbleweed.",
  "modified": "2025-05-07T18:09:00.963701Z",
  "published": "2024-06-15T00:00:00Z",
  "related": [
    "CVE-2014-3589",
    "CVE-2014-3598",
    "CVE-2016-0740",
    "CVE-2016-0775"
  ],
  "upstream": [
    "CVE-2014-3589",
    "CVE-2014-3598",
    "CVE-2016-0740",
    "CVE-2016-0775"
  ],
  "references": [
    {
      "type": "ADVISORY"
    },
    {
      "type": "WEB",
      "url": "https://www.suse.com/security/cve/CVE-2014-3589"
    },
    {
      "type": "WEB",
      "url": "https://www.suse.com/security/cve/CVE-2014-3598"
    },
    {
      "type": "WEB",
      "url": "https://www.suse.com/security/cve/CVE-2016-0740"
    },
    {
      "type": "WEB",
      "url": "https://www.suse.com/security/cve/CVE-2016-0775"
    }
  ],
  "affected": [
    {
      "package": {
        "name": "python-Pillow",
        "ecosystem": "openSUSE:Tumbleweed",
        "purl": "pkg:rpm/opensuse/python-Pillow&distro=openSUSE%20Tumbleweed"
      },
      "ranges": [
        {
          "type": "ECOSYSTEM",
          "events": [
            {
              "introduced": "0"
            },
            {
              "fixed": "3.1.1-4.1"
            }
          ]
        }
      ],
      "ecosystem_specific": {
        "binaries": [
          {
            "python-Pillow-tk": "3.1.1-4.1",
            "python-Pillow": "3.1.1-4.1"
          }
        ]
      },
      "database_specific": {
        "source": "https://ftp.suse.com/pub/projects/security/osv/openSUSE-SU-2024:10567-1.json"
      },
      "versions": []
    }
  ],
  "schema_version": "1.6.0"
}